skycatalogs/objects/gaia_object.py

```diff
@@ -143,6 +141,15 @@ def get_config(cls):
     @ignore_erfa_warnings
     @staticmethod
     def load_collection(region, skycatalog, mjd=None):
+        '''
+        region      One of Disk, PolygonalRegion from skyCatalogs.utils.shapes.
+                    Box is not currently supported
+        skycatalog  An instance of the SkyCatalog class
+        mjd         Time at which objects are to be assembled. Ignored for
+                    Gaia stars
+        '''
+        if not skycatalog:
```

Comment:
I don't think this test is implemented correctly or appropriate, but I guess it is largely harmless if you refuse to remove it. This is essentially type checking (though not actually doing it). If you want type checking, then it really should be applied uniformly across the interfaces.   It only came up here because changes you made broke the existing implementation.